hopp/simulation/technologies/wind/pysam_wind.py

```diff
@@ -0,0 +1,65 @@
+from attrs import define, field
+
+import PySAM.Windpower as Windpower
+
+from hopp.simulation.base import BaseClass
+from hopp.simulation.technologies.sites import SiteInfo
+from hopp.type_dec import resource_file_converter
+from hopp.utilities.utilities import load_yaml
+
+
+@define
+class PySAMWind(BaseClass):
+    config_dict: dict = field(converter=dict)
+    site: SiteInfo = field()
+    timestep: tuple = field(default=(), converter=tuple)
+
+    system_model: Windpower = field(init=False)
+
+    def __attrs_post_init__(self):
+        input_file_path = resource_file_converter(self.config_dict["simulation_input_file"])
```

Comment:
I'm seeing this yaml file loading code used a few times (and we'll definitely be doing this for other classes), might be worth moving into its own utility function in `hopp.utilities.utilities`

Reply:
agreed

Reply:
Sounds good.